tests/compile-pass/non_inferred_generic_types_for_box.rs

```diff
@@ -0,0 +1,9 @@
+use auto_impl::auto_impl;
+
+#[auto_impl(Box)]
+trait Foo {
+    fn foo<T>();
+    fn bar<U>(&self);
+    fn baz<V>(&mut self);
+    fn qux<W>(self);
+}
```

Comment:
Could you add a newline here? As a general hint: it's usually expected that all files end with a single trailing newline. You should probably configure your editor to automatically insert one. While doing that you should also configure your editor to automatically remove any trailing whitespace (whitespace at the end of a line). That's also something most people don't want in their code. Thanks! :)